### PR TITLE
7za: add page

### PR DESCRIPTION
**Pull request by joelhy**
add 7zip command line utility 7za to common page

### CODE REVIEW

**Comment by joelhy** [comment]
origin from https://github.com/tldr-pages/tldr/pull/486

File: pages/common/7za.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# 7za
+
+> A file archiver with highest compression ratio
```
Comment (waldyrious) [inline pages/common/7za.md]: "highest" is a bit unclear here -- highest than anything else currently available? highest possible? I would suggest replacing with just "high".

File: pages/common/7za.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- package and compress directory or file
```
Comment (waldyrious) [inline pages/common/7za.md]: **a** directory

File: pages/common/7za.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- decompress an existing 7z file
+
+`7za x {{compressed.7z}}`
+
+- compress to zip format
+
+`7za a -tzip {{compressed.zip}} {{directory_or_file_to_compress}}`
+
+- create multipart 7zip file; part_size: specifies part size in Bytes, Kilobytes, Megabytes or Gigabytes.
```
Comment (waldyrious) [inline pages/common/7za.md]: the ":" is unnecessary here. Also maybe wrap "part_size" in code tags `` or the token syntax {{}} to make it explicit that it refers to the command parameter below.

File: pages/common/7za.md (hunk 2 above)
Comment (joelhy) [inline pages/common/7za.md]: Fixed. Thanks a lot.

File: pages/common/7za.md (hunk 3 above)
Comment (joelhy) [inline pages/common/7za.md]: Wording changed and code tags `` added.

**Comment by rubenvereecken** [comment]
I've got a paranoia with compression tools about them keeping directory structure. I think it started in the Windows days with zip and the like where I often ended up with everything dumped in the current directory (extract zipped directory to this directory) or, in an effort to battle this behavior, ended up with a two-layered directory `unzipped/unzipped/...`.

Would it be useful to add "with original directory structure" to the decompress example, [like unrar did](https://github.com/tldr-pages/tldr/pull/592/files#diff-569a49e2a10656da462089e7bb76e82cR9)?

**Comment by joelhy** [comment]
"with original directory structure" added to decompress comment.

**Comment by rubenvereecken** [comment]
I guess it doesn't hurt. Thanks for caring for my paranoia. You have my :+1:

**Comment by waldyrious** [comment]
:+1:

**Pull request merged**